iset.mm

```diff
@@ -49989,6 +49989,21 @@ is the image of an element of its domain (see ~ foima ).  (Contributed
       $.
   $}
 
+  ${
+    $d A x y $.  $d F x y $.  $d ph y $.
+    f1ofni.f $e |- ( ph -> ( F |` A ) Fn A ) $.
+    f1ofni.i $e |- ( ph -> A. x e. A ( F ` x ) = x ) $.
+    $( A function which maps sets to themselves is one-to-one and onto.
+       (Contributed by Jim Kingdon, 25-Aug-2022.) $)
+    f1ofni $p |- ( ph -> ( F |` A ) : A -1-1-onto-> A ) $=
```

Comment:
It looks like the statement is trying hard to hide the fact that F restricted to A is the identity.  It would be more natural to have the conclusion ``( F |` A ) = ( _I |` A ) ``.  If you need the precise statement of your conclusion, there is probably a theorem somewhere `` ( _I |` A ) : A -1-1-oneto-> A`` from which you conclude what you need.

Reply:
> It looks like the statement is trying hard to hide the fact that F restricted to A is the identity. It would be more natural to have the conclusion `` ( F |` A ) = ( _I |` A ) ``. If you need the precise statement of your conclusion, there is probably a theorem somewhere `` ( _I |` A ) : A -1-1-oneto-> A `` from which you conclude what you need.

I'll just delete this theorem. I didn't end up using it.